packages/web/.env/.env.prod

```diff
@@ -1,95 +1,3 @@
-VITE_REWARDS_MANAGER_TOKEN_PDA=3V9opXNpHmPPymKeq7CYD8wWMH8wzFXmqEkNdzfsZhYq
-VITE_PAYMENT_ROUTER_PROGRAM_ID=paytYpX3LPN98TAeen6bFFeraGSuWnomZmCXjAsoqPa
-
-VITE_WORMHOLE_RPC_HOSTS=https://wormhole-v2-mainnet-api.certus.one,https://wormhole.inotel.ro,https://wormhole-v2-mainnet-api.mcf.rocks,https://wormhole-v2-mainnet-api.chainlayer.network,https://wormhole-v2-mainnet-api.staking.fund,https://wormhole-v2-mainnet-api.chainlayer.network
-VITE_ETH_BRIDGE_ADDRESS=0x98f3c9e6E3fAce36bAAd05FE09d375Ef1464288B
-VITE_ETH_TOKEN_BRIDGE_ADDRESS=0x3ee18B2214AFF97000D974cf647E7C347E8fa585
-VITE_SOL_BRIDGE_ADDRESS=worm2ZoG2kUd4vFXhvjh93UUH596ayRfgQ2MgjNMTth
-VITE_SOL_TOKEN_BRIDGE_ADDRESS=wormDTUJ6AWPNvk59vGQbDvGJmqbDTdgWgAqcLBCgUb
-
-VITE_METADATA_PROGRAM_ID=metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s
-
-VITE_GA_MEASUREMENT_ID=G-V6N1ZTVGS5
-
-VITE_PUBLIC_PROTOCOL=https:
-VITE_PUBLIC_HOSTNAME=audius.co
-VITE_GA_HOSTNAME=audius.co
-VITE_REACHABILITY_URL=https://audius.co/204
-VITE_NOTIFICATION_URL=https://download.audius.co/whatsnew
-
-VITE_SCHEME=audius
-
-VITE_EXPLORE_CONTENT_URL=https://download.audius.co/static-resources/explore-content.json
-VITE_SUGGESTED_FOLLOW_HANDLES=https://download.audius.co/static-resources/signup-follows.json
-VITE_EAGER_DISCOVERY_NODES=https://discoveryprovider.audius.co,https://discoveryprovider2.audius.co,https://discoveryprovider3.audius.co
-
-VITE_SENTRY_DSN=https://e26d2b01488f4b538ea235de6466c40a@s.audius.co/1457231
-
-VITE_SEGMENT_WEB=mGyaUaZz6IZwAM6taytNhubjCX4RPR0x
-VITE_SEGMENT_MOBILE=AbVxGDZBuVRCAVuhRaoF3cNsOB3vUCos
-VITE_SEGMENT_ELECTRON=eMGe35ghRLWsEWX9EpzblQInDH1CmzAa
-VITE_AMPLITUDE_API_KEY=86760558b8bb1b3aae61656efd4ddacb
-VITE_AMPLITUDE_PROXY=gain.audius.co
-VITE_ADROLL_AVD_ID=SDJ23TS7DBFPLMXRDF6JQY
-VITE_ADROLL_PIX_ID=JZSZK5GWHBFL3FQ3NRFPNY
-VITE_FCM_PUSH_PUBLIC_KEY=BDBRR9S_UczQBflfaFPQf1JuCIZhMk90G_QnRI9wWuoObt7Ck6sRCxPDJfB_mJY3p55WdAO4C4xVJTAMoiv6JJQ
-VITE_SAFARI_WEB_PUSH_ID=web.co.audius
-
-VITE_OPTIMIZELY_KEY=DAJbGEJBC21dzFRPv8snxs
-VITE_INSTAGRAM_APP_ID=189700309435573
-VITE_INSTAGRAM_REDIRECT_URL=https://audius.co/
-
-GENERATE_SOURCEMAP=false
-
-VITE_RECAPTCHA_SITE_KEY=6LfVR-0ZAAAAADFcqNM1P1IafKwQwN0E_l-gxQ9q
-VITE_HCAPTCHA_SITE_KEY=b250803e-dcba-428c-bc87-8acf559aacb9
-
-VITE_BITSKI_CLIENT_ID=661ce11a-3e0f-4659-b365-795ad2111f42
-VITE_BITSKI_CALLBACK_URL=https://audius.co/bitski-callback.html
-
-VITE_OPENSEA_API_URL=https://collectibles.audius.co/api/v1
-VITE_COGNITO_KEY=live_publishable_key_9afb0a1e2e32ba707a70e66c6d309500
-VITE_COGNITO_TEMPLATE_ID=flwtmp_eMMe8VQGrKvLCX
-
-VITE_FINGERPRINT_PUBLIC_API_KEY=MNtDQ4NCsNSP7YOkOiQT
-VITE_FINGERPRINT_ENDPOINT=https://fp.audius.co
-
-VITE_OTEL_TRACING_ENABLED=true
-
-VITE_STRIPE_CLIENT_PUBLISHABLE_KEY=pk_live_51LPsGuCJOWtpH6AEKshlCs3L8QhAfevNvhev8K9a0u92O5ku83KRjLIqCdxgf3NhitdtmMGlw0Wjf33NjZJjZUBz006A3IoSiQ
-
-VITE_USE_HASH_ROUTING=false
-
-VITE_COINFLOW_MERCHANT_ID=tikilabs
-VITE_COINFLOW_APP_ID=9JBW2RHC7JNJN8ZQ
\ No newline at end of file
+GENERATE_SOURCEMAP=false
```

Comment:
Was it not possible to put these in the env object too? Should be able to reference it from `vite.config.ts` I would think

Reply:
Let me grab you to chat about this tmw. Merging PR, but will open a subsequent one

Reply:
Yeah let's chat today! I think we missed updating the env vars in `index.html` too https://github.com/AudiusProject/audius-protocol/blob/main/packages/web/index.html#L167